docs/data_pack.md

```diff
@@ -1,5 +1,108 @@
 # DataPack Tutorial
 
+## Get data
+`DataPack.get_data()` is commonly used to retrieve data from a `DataPack`. This method returns a generator that generates dictionaries containing data requested, and each dictionary has a scope that covers certain range of data in the `DataPack`.
+To understand this, let's consider a dummy case.  Given that there is a document in the `DataPack` instance `data_pack`, we want to get the full document in `data_pack`.
+We can set up the `data_pack` using the following code under forte project root path.
+```python
```

Comment:
I am wondering if this can be used as a notebook?

Reply:
Do you mean jupyter notebook? I think so, it will also be easier to read the output.